web/frontend/src/types/overrides.ts

```diff
@@ -49,6 +49,7 @@ export interface SLAMOverrides {
   viewMap: (sessionId: string) => void;
   validateMapName: (mapName: string) => string;
   mappingDetails: MappingDetails;
+  slamModel: string;
```

Comment:
`slamModel` is `''` with App in production.

Reply:
actually just realized we were adding the slamModel as part of this PR, which I don't think we need to for this behavior. I don't think we need to do this for this pr, but if you need it for another pr then its fine to add.

Reply:
Removed it. Instead, this PR has changes that allow us to use `Properties()` for all SLAM libraries: https://github.com/viamrobotics/app/pull/4194